core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -94,7 +119,39 @@ oppia.directive('audioControls', [
           };
 
           var playPauseAudioTranslation = function(languageCode) {
-            $scope.extraAudioControlsAreShown = true;
+            if (AudioTranslationManagerService
+              .isAutogeneratedLanguageCodeSelected()) {
+              playPauseAutogeneratedAudioTranslation();
+            } else {
+              playPauseUploadedAudioTranslation(languageCode);
+            }
+          };
+
+          var playPauseAutogeneratedAudioTranslation = function() {
+            $scope.audioSettingsButtonIsShown = true;
+            $scope.rewindButtonIsShown = false;
+            // SpeechSynthesis in Chrome seems to have a bug
+            // where if you pause the utterance, wait for around
+            // 15 or more seconds, then try resuming, nothing
+            // will sound. As a temporary fix, just restart the
+            // utterance from the beginning instead of resuming.
+            if (AutogeneratedAudioPlayerService.isPlaying()) {
+              AutogeneratedAudioPlayerService.cancel();
+            } else {
+              AutogeneratedAudioPlayerService.play(
+                $scope.getContentHtml(),
```

Comment:
Mm -- I think you need to send this through a helper service that transforms html to text. E.g. for links, the link text should be read, rather than the link being skipped altogether.

Reply:
I think it seems a bit easier to just let the speech chunker service take the html and handle it?